lib/routes/admin-api/feature.js

```diff
@@ -52,6 +61,42 @@ class FeatureController extends Controller {
         }
     }
 
+    async listTags(req, res) {
+        const name = req.params.featureName;
+        const tags = await this.featureTagStore.getAllTagsForFeature(name);
+        res.json({ version, tags });
+    }
+
+    async addTag(req, res) {
+        const { featureName } = req.params;
+        const { value, type } = req.body;
+        const userName = extractUser(req);
+        try {
+            await nameSchema.validateAsync({ name: featureName });
+            await tagSchema.validateAsync({ value, type });
```

Comment:
I would probably change this line to:

`const data = await tagSchema.validateAsync(req.body);`

Reply:
Yup, agreed.

Reply:
Solved in 8def623